.env.example

```
@@ -8,6 +8,9 @@ DATABASE_URL="mysql://emodex:emodex@localhost:3307/emodex"
 OPENAI_API_KEY=""
 OPENAI_MODEL="openai/gpt-5-codex"
 
+# Blob storage configuration
+BLOB_READ_WRITE_TOKEN=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)